PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -35,9 +35,39 @@ def parse_arguments():
     return args
 
 
+def get_sets_by_category(mb_core,category_name):
+    """
+    Identify entity sets in the given geometry based on a category tag value.
+
+    Input:
+    ______
+       mb_core: Core
+           A PyMOAB core instance with a loaded data file.
```

Comment:
Nice work being specific about the core needing a loaded file. It would be an easy thing to leave out.